### PR TITLE
Fix GUI layout issues on Windows

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

**Comment by pcanal** [comment]
Could you give some hints in the commit log of the kind of layout issue being addressed?  

What is the "value" of keeping the old code (behind ```#if 0```) in this case?

**Comment by bellenot** [comment]
> Could you give some hints in the commit log of the kind of layout issue being addressed?
Well, in some cases, the GUI layout is wrong. Do you want me to take screenshots?

> What is the "value" of keeping the old code (behind `#if 0`) in this case?
Form my own purpose, I want to test on a machine with dual screens once I can access it

File: graf2d/win32gdk/gdk/src/gdk/win32/gdkwindow-win32.c [hunk 1]
```diff
@@ -58,6 +59,7 @@ SafeAdjustWindowRectEx(RECT * lpRect,
       lpRect->bottom -= lpRect->top;
       lpRect->top = 0;
    }
+#endif
```
Comment (eguiraud) [inline graf2d/win32gdk/gdk/src/gdk/win32/gdkwindow-win32.c:62]: will this code never be needed? (i.e. should it be removed?)

**Comment by bellenot** [comment]
@pcanal  I found a specific example use case and updated the commit log

**Comment by pcanal** [comment]
> Well, in some cases, the GUI layout is wrong. Do you want me to take screenshots?

No but a short descriptions of the elements used and/or the kind of situation would be helpful.

> Form my own purpose, I want to test on a machine with dual screens once I can access it (and I'll remove it once tested)

Can you add a comment along this line next to the code? (This would prevent somebody else from (proposing to) removing it.

**Comment by pcanal** [comment]
> I found a specific use case as example and updated the commit log

My github refuses to show me the updated commit log.

**Comment by bellenot** [comment]
[Here it is](https://github.com/root-project/root/pull/6077#issue-456262469)

**Comment by pcanal** [comment]
> Here it is

Could you **also** make it part of the commit log (finding the PR summary from a given commit is not that "easy")? 

Thanks.

**Comment by bellenot** [comment]
No idea what you're talking about (I was sure my edit would be part of the commit log...) But I'll figure out

**Comment by bellenot** [comment]
> > Here it is
> 
> Great. Could you **also** make it part of the commit log (finding the PR summary from a given commit is not that "easy")?
> 
> Thanks.

Done

**Pull request merged**